starters/express-typescript-typeorm-postgres/src/utils/log-helper.spec.ts

```diff
@@ -0,0 +1,153 @@
+describe(`LogHelper`, () => {
+  let consoleSpy;
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe(`log method`, () => {
```

Comment:
The tests are very repetitive we could simplify it using the `describe.each()` feature
```ts
describe.each([
      ['log', 'INFO'],
      ['info', 'INFO'],
      ['warn', 'WARN'],
      ['debug', 'DEBUG'],
      ['error', 'ERROR'],
    ])('%s function', (method, level) => {
      test(`calls logger (console by default) if ${level} is set in process.env.ALLOWED_LOG_LEVELS`, () => {
        process.env.ALLOWED_LOG_LEVELS = 'INFO,WARN,DEBUG,ERROR';
        logger[method]('Test');
        expect(console[method]).toHaveBeenCalledWith(`[${level}]`, 'Test');
      });

      test(`does NOT call logger (console by default) if level is NOT in process.env.ALLOWED_LOG_LEVELS`, () => {
        process.env.ALLOWED_LOG_LEVELS = '';
        logger[method]('Test');
        expect(console[method]).not.toHaveBeenCalled();
      });
    });
```

Reply:
I am generally not fond of these types of test abstractions. I know we could do that, but whenever something later gets customised it is a pain to fix these abstractions.

That said, I'll modify the tests as per your suggestions, because it is not likely these are going to be broken later.